ui/public/i18n/en/translation.json

```diff
@@ -1,7 +1,8 @@
 {
   "common": {
     "required": "Required",
-    "work_in_progress": "Work in progress"
+    "work_in_progress": "Work in progress",
+    "eg_value": "E.g. {value}"
```

Comment:
![image](https://github.com/NethServer/ns8-nextcloud/assets/3164851/e304495e-00d1-4999-90f0-60e206fd0ce8)


I did it a bit differently, we cannot send to translation a hostname, it could be changed by the translators